src/lib/components/wizard.component.spec.ts

```diff
@@ -79,9 +79,15 @@ describe('WizardComponent', () => {
     expect(wizardTestFixture.debugElement.query(By.css('aw-wizard > :first-child')).name).toBe('aw-wizard-navigation-bar');
     expect(wizardTestFixture.debugElement.query(By.css('aw-wizard > :last-child')).name).toBe('div');
 
-    expect(navBarEl.classes).toEqual({ 'horizontal': true, 'small': true });
-    expect(wizardEl.classes).toEqual({ 'horizontal': true });
-    expect(wizardStepsDiv.classes).toEqual({ 'wizard-steps': true, 'horizontal': true });
+    expect(
+      Object.keys(navBarEl.classes).filter(m => navBarEl.classes[m] === true)
```

Comment:
I think it makes sense to move the pattern `Object.keys(element.classes).filter(class => element.classes[class] === true)` to a new test utils method

Reply:
Sure.